src/components/ParticipantInfo/ParticipantInfo.tsx

```diff
@@ -128,11 +128,10 @@ const useStyles = makeStyles((theme: Theme) =>
     cursorPointer: {
       cursor: 'pointer',
     },
-    dominantSpeaker: {
-      border: `solid ${borderWidth}px #7BEAA5`,
-      margin: `${GRID_MODE_MARGIN} - borderWidth`,
-    },
-    mobileGridMode: {
+    gridView: {
+      background: '#000000',
+      border: `${theme.participantBorderWidth}px solid transparent`,
```

Comment:
Hmm, not sure how I didn't see this before, but it looks like there's a black border around the participants:
![image](https://user-images.githubusercontent.com/12685223/172960341-e3829679-0b21-482a-8e33-aa9b2b6161fe.png)

I think it's really just a transparent border that's showing the black background. So I guess we just make the border the same color as the grid background color here?

Also I think `background: #000000` can be removed here. Looks like there's already `background: 'black'` in the `container` class.